### PR TITLE
[Database] Allow falsy values to batch()

### PR DESCRIPTION
**Pull request by radex**
[Database] You can now pass falsy values to `Database.batch(...)` (false, null, undefined). This is
     useful in keeping code clean when doing operations conditionally

### CODE REVIEW

File: src/Database/index.js [hunk 1]
```diff
@@ -43,24 +43,29 @@ export default class Database {
 
   // Executes multiple prepared operations
   // (made with `collection.prepareCreate` and `record.prepareUpdate`)
-  async batch(...records: $ReadOnlyArray<Model>): Promise<void> {
+  // Note: falsy values (null, undefined, false) passed to batch are just ignored
```
Comment (kokusGr) [inline src/Database/index.js]: I think this also should be in docs/changelog :)

Comment (radex) [reply src/Database/index.js]: done

Comment (kokusGr) [reply src/Database/index.js]: I meant info that falsy values are ignored but info about `model.batch` is also good :)

Comment (radex) [reply src/Database/index.js]: fixed on master https://github.com/Nozbe/WatermelonDB/commit/f8922d2ba11b29d7db744d2071feda6a55c7a292

**Comment by kokusGr** [comment]
Ok, thanks! :)

**Pull request merged**